docs/tutorials/fine_calibrations.ipynb

```diff
@@ -53,6 +54,16 @@
     "qubit = 0"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "5b924bfd",
+   "metadata": {},
+   "source": [
+    "### Instruction schedule map\n",
+    "\n",
+    "We will run the fine calibration experiments with our own pulse schedules. To do this we create an instruction to schedule map which we populate with the schedules we wish to work with. This instruction schedule map is then given to the transpile options of the calibration eperiments so that the Qiskit transpiler can attach the pulse schedules to the gates in the experiments. We will base all our pulses on the default `X` pulse of Armonk."
```

Comment:
```suggestion
    "We will run the fine calibration experiments with our own pulse schedules. To do this we create an instruction to schedule map which we populate with the schedules we wish to work with. This instruction schedule map is then given to the transpile options of the calibration experiments so that the Qiskit transpiler can attach the pulse schedules to the gates in the experiments. We will base all our pulses on the default `X` pulse of Armonk."
```